src/elements/content-picker/ItemList.js

```diff
@@ -120,7 +120,7 @@ const ItemList = ({
                             height={height}
                             disableHeader
                             headerHeight={0}
-                            rowHeight={50}
+                            rowHeight={65}
```

Comment:
Does this still look okay in wider resolutions and/or with single-line items? Can you post a screenshot?

Reply:
Added screenshots to the PR description.